src/Symfony/Bridge/Twig/UndefinedCallableHandler.php

```diff
@@ -64,34 +68,40 @@ class UndefinedCallableHandler
         'workflow' => 'enable "framework.workflows"',
     ];
 
-    public static function onUndefinedFilter(string $name): bool
+    /**
+     * @return TwigFilter|false
+     */
+    public static function onUndefinedFilter(string $name)
     {
         if (!isset(self::FILTER_COMPONENTS[$name])) {
             return false;
         }
 
-        self::onUndefined($name, 'filter', self::FILTER_COMPONENTS[$name]);
-
-        return true;
+        throw new SyntaxError(self::onUndefined($name, 'filter', self::FILTER_COMPONENTS[$name]));
     }
 
-    public static function onUndefinedFunction(string $name): bool
+    /**
+     * @return TwigFunction|false
+     */
+    public static function onUndefinedFunction(string $name)
     {
         if (!isset(self::FUNCTION_COMPONENTS[$name])) {
             return false;
         }
 
-        self::onUndefined($name, 'function', self::FUNCTION_COMPONENTS[$name]);
+        if ('webpack-encore-bundle' === self::FUNCTION_COMPONENTS[$name]) {
+            return new TwigFunction($name, static function () { return ''; });
```

Comment:
you could extract the nop function as a static local variable for a bit of performance and readability 
`static $nopFunction = static function () { return ''; };`

Reply:
I don't think that'd make any difference